hypothesis-python/docs/strategies.rst

```diff
@@ -152,8 +152,27 @@ and then tell ``setuptools`` that this is your ``"hypothesis"`` entry point:
 
     # setup.py
 
-    ...
+    # You can list a module to import by dotted name
+    entry_points = {"hypothesis": ["_ = mymodule.a_submodule"]}
+
+    # Or name a specific function too, and Hypothesis will call it for you
     entry_points = {"hypothesis": ["_ = mymodule:_hypothesis_setup_hook"]}
-    ...
 
 And that's all it takes!
+
+
+Interaction with :pypi:`pytest-cov`
```

Comment:
👍

Reply:
Can you cherry-pick and try out https://github.com/Zac-HD/hypothesis/commit/3503e3a4e4f9d0af45812de58b1eef9c582b7145?  The idea is that deferring imports for long enough in our pytest plugin would mean that Hypothesis entry points are only loaded after pytest has loaded all their entrypoints, but if it doesn't work for your workload it's *definitely* not worth the trouble.